addons/gitlab/tests/test_views.py

```diff
@@ -518,24 +519,94 @@ def test_link_repo_non_existent(self, mock_repo):
 
         assert res.status_code == 400
 
+    user = 'octo-cat'
```

Comment:
nitpick: if this is constant fixture data, can you capitalize the variable names and move it either to the top of the class definition or the file definition (whichever is more appropriate)?